app/views/dashboard/users.html.haml

```diff
@@ -0,0 +1,17 @@
+- @data.each do |office_name, users|
+  %h2= office_name
+
+  %table
+    %thead
+      %tr
+        %th Name
+        %th Email
+        %th Role
+        -# %th Actions
+    %tbody
+      - users.each do |user|
+        %tr
+          %td= "#{user.first_name} #{user.last_name}"
```

Comment:
```suggestion
          %td= "#{user.to_s}”
```

Reply:
`to_s` isn't needed when you're interpolating - could just be ` %td= "#{user}”` or ` %td= user.to_s`

Reply:
True and fixed